src/genie/libs/parser/junos/show_security.py

```diff
@@ -0,0 +1,112 @@
+"""show_security.py
+JunOS parsers for the following show commands:
+    * show security policies hit-count
+"""
+# Python
+import re
+
+# Metaparser
+from genie.metaparser import MetaParser
+from pyats.utils.exceptions import SchemaError
+from genie.metaparser.util.schemaengine import (Any, Optional, Use, Schema, ListOf)
```

Comment:
I don't think the import list needs to be wrapped in brackets. Additionally, I think some of these imports are unused.